Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php

```diff
@@ -204,7 +323,6 @@ public function method444f1()
     }
 
     /**/
-
```

Comment:
> This fixer is broken if it removes that space.

Which space is removed? Again: In input there is no newline after the comment. And the fixer should not add a newline. This is fixed by this pr, so the test has to be adjusted.